src/vs/workbench/workbench.common.main.ts

```diff
@@ -117,7 +117,7 @@ import 'vs/editor/common/services/treeViewsDndService';
 import 'vs/workbench/services/textMate/browser/textMateTokenizationFeature.contribution';
 import 'vs/workbench/services/userActivity/common/userActivityService';
 import 'vs/workbench/services/userActivity/browser/userActivityBrowser';
-import 'vs/workbench/services/issue/browser/issueTroubleshoot';
+import 'vs/workbench/contrib/issue/browser/issueTroubleshoot';
```

Comment:
Since this is a `contrib`, it should move below to the `//#region --- workbench contributions` section.

And to confirm: this is a contribution meant for both web and desktop?